README.md

```diff
@@ -973,6 +972,61 @@ Learn more about class-transformer and how to handle more complex object constru
 This behaviour is enabled by default.
 If you want to disable it simply pass `useClassTransformer: false` to createExpressServer method.
 
+## Auto validating action params
+
+Sometimes parsing a json object into instance of some class is not enough. 
+E.g. `class-transformer` doesn't check whether the property's types are correct, so you can get runtime error if you rely on TypeScript type safe. Also you may want to validate the object to check e.g. whether the password string is long enough or entered e-mail is correct.
+
+It can be done easily thanks to integration with [class-transformer-validator][7] which internally depends on [class-validator][6]. So first of all, you have to install the modules:

```

Comment:
`class-transformer-validator` isn't tightly coupled with `routing-controllers`, it can work with anything, eg. with pure express.js:
```typescript
async (req, res) => {
    try {
        const post = await transformAndValidate(Post, req.body);
        await Post.persist(post);
        res.send(`Post ${post.getFullName()} has been added!`);
    } catch (error) {
        res.status(400).json(error);
    }
}
```
Or even for loading config JSON:
```typescript
try {
    const jsonString = await fs.readFileAsync("./files/config.json", "utf8");
    const config = await transformAndValidate(Config, jsonString);
} catch (error) {
    console.err(error);
}
```

Of course, it's just a simple wrapper which "simply call two methods of two libraries in sequence" and you could do this by its own. But if you try to make the code clean and keep SOLID, due to Single responsibility principle you should move that part of code into a new function (and make some tests?) which already has been done in `class-transformer-validator` 😉

It's not a 100KB library with 50 deps from which we need 1 function. I think that we should DRY and don't make copy-paste of every lib just to avoid dependency.